### ISSUE TITLE
[Trade Republic] Support für passwortgeschütze PDFs hinzufügen

### ISSUE THREAD

**Issue by TebosBrime** [tresorone/tresor-import#3214]
Es ist mitlerweile möglich bei TR einen Zweitschriftenversand einzureichten. Durch diesen bekommt man jede Abrechnung per E-Mail. Die Dokumente sind allerdings per Passwort geschützt. (Dieses bekommt man vor der ersten Abrechnung). Dies sorgt allerdings dafür, dass ein Import nicht möglich ist. (Außer man entfernt das Passwort manuell, welches man mit Adobe nur in der kostenpflichtigen Version kann)

Ich wünsche mir ein Feature, dass auch diese PDFs verwendet werden können. Eine beispielhafte Umsetzung wäre ein PopUp, welches nach einem Passwort fragt. Dies sollte auch im js möglich sein. 

Im Zweifelsfall könnte ich zum Test eine PDF zur Verfügung stellen, es sollte aber auch reichen eine eigene PDF mit einem Passwort zu versehen. 

https://helpx.adobe.com/de/acrobat/using/securing-pdfs-passwords.html
https://support.traderepublic.com/de-de/120-Ist-es-m%C3%B6glich-einen-Zweitschriftenversand-einzurichten

**Comment by dredav** [comment]
Oh das mit dem Zweitschriftenversand ist mir neu. Haste dich selbst als Firma angegeben oder den Abschnitt komplett weggelassen...?

**Comment by TebosBrime** [comment]
Nein, dass ist nicht nötig. Habe nach Anlage 2.3 (Sonderbedingungen Postbox) Punkt 1.2 um Mailversand gebeten. Das Template hab ich nicht genutzt. 

Habe mir aber nun eine Software geschrieben, welche mir die files autmatisch entschlüsselt zu sendet. Ich warte ich nur noch auf meine DSGVO Auskunft um die files der letzten 15 Moante zu bekommen.

**Comment by dredav** [comment]
Sehr spannend. Danke für den Hinweis. Ich werden prüfen ob und wie wir das ganze in tresor.one / parqet bzw. dem Import implementieren können.
